### DISCUSSION TITLE
Limit image size?

### DISCUSSION THREAD

**Discussion by stuartcusackie** [area17/twill#892] [Q&A]
How would one go about limiting an image size on the default image block? I can see that the 'crops' array allows us to define minValues, but there doesn't seem to be a way to limit or resize a large image.

**Comment by dvolkering** [comment]
Is there any option to resize images to maximum values instead of using "crops" from photos?

Also, the uploaded images by the media browser have a bigger file size compared to the original images. Original upload is 130kb, but when I want to access the default crop, it's 220kb. That's almost double the filesize.
